astroquery/mast/tests/test_mast_remote.py

```diff
@@ -1016,6 +1021,15 @@ def test_tesscut_get_cutouts_mt(self):
                                      moving_target=True)
             assert error_tica_mt in str(error_msg.value)
 
+    @pytest.mark.xfail(raises=InputWarning)
```

Comment:
This is not the correct approach, xfails are more for cases of known buggy behaviour, or expected server side issues, etc.
Instead, please use pytest.warns context manager, there are plenty of usage examples for it in astroquery.

Reply:
The reason I had to do it this way is because running with pytest.warns will allow the query to run fully, which will take several minutes and also end up failing the test with the timeout error. This is for the queries that exceed the recommended timeout limit of 600 seconds because the cutout size is too large and the `timeout` arg hasn't been changed.